ReferenceBackendAPI.md

```diff
@@ -0,0 +1,193 @@
+# Reference Backend API
+
+## HTTP Method
+
+The reference backend is deployed to [Twilio Functions](https://www.twilio.com/docs/runtime/functions), a serverless environment.
+
+Twilio Functions does not utilize HTTP method so you can just use `POST` for all requests.
+
+## Authentication
+
+All requests must set the `Authorization` header to the correct [passcode](README.md#deploy-the-app-to-twilio). The backend will validate the passcode before processing each request.
+
+## Errors
+
+All error responses have this shape:
+
+```json
+{
+  "error": {
+    "message": "error creating room",
+    "explanation": "Room already exists."
+  }
+}
+```
+
+## Endpoints
+
+### /verify-passcode
+
+Verifies that the passcode in the request header is correct. The client can call this endpoint to validate a passcode the user has entered.
+
+Request parameters:
+
+```
+None since the passcode to verify is in the request header
+```
+
+Example response:
+
+```json
+{
+  "verified": true
+}
+```
+
+### /create-stream
+
+This endpoint creates a Twilio video room, Twilio Live player streamer, Twilio Live media processor, Twilio sync service, and Twilio conversation. The user is added to the conversation and some user state is updated in sync objects. The endpoint returns a [Twilio access token](https://www.twilio.com/docs/iam/access-tokens). The token includes grants that allow the client to connect to the video room, sync objects, and conversation. The user that creates the stream is the host for that stream.
```

Comment:
```suggestion
This endpoint creates a Twilio video room, Twilio Live player streamer, Twilio Live media processor, Twilio sync service, and Twilio conversation. The user is added to the conversation and some user state is updated in sync objects. The endpoint returns a [Twilio access token](https://www.twilio.com/docs/iam/access-tokens). The token includes grants that allow the client to connect to the Twilio Video SDK, Twilio Sync SDK, and Twilio Conversations SDK. The user that creates the stream is the host for that stream.
```
What do you think about this? Just to be more specific about how the token is used.